ci/tasks/create-pull-request.yml

```diff
@@ -0,0 +1,14 @@
+---
+platform: linux
+inputs:
+  - name: git-repo
```

Comment:
I'm wondering if this need to be a different input - e.g. the docker git resource in this case. Which makes it less generic somehow.

Reply:
For this task, it only uses the `git-repo` to run the `create-pull-request.sh`. It's not actually used anywhere in the script itself so we can stick to using the regular `git-repo` and not the docker-specific one.